CHANGELOG.md

```diff
@@ -8,8 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Updated
-- Improve performance of single-argument methods by using an outer Array instead
-  of a Hash
+- Improve performance of both zero-argument and single-argument methods by using
+  an outer Array instead of a Hash
```

Comment:
Nit: I'd split these into two CHANGELOG entries personally, to make it easier to connect to separate changes in the history, but weakly held